concordia/models.py

```diff
@@ -21,6 +21,18 @@ class UserProfile(MetricsModelMixin("userprofile"), models.Model):
     user = models.ForeignKey(User, on_delete=models.CASCADE)
 
 
+class OverlayPosition(object):
+    """
+    Used in carousel slide content management
+    """
+
+    LEFT = "left"
```

Comment:
Do we intend to use these constants later? We don't seem to have any code other than line 32 which uses them.